src/mmio_types/keys.rs

```diff
@@ -52,8 +91,19 @@ impl Keys {
       0
     }
   }
+
+  pub fn any_pressed(self, key_mask: u16) -> bool {
+    self.0 & key_mask != 0
+  }
+
+  pub fn pressed(self, key: Key) -> bool {
+    self.any_pressed(key as u16)
+  }
+
+  pub fn released(self, key: Key) -> bool {
+    self.0 & (key as u16) == 0
+  }
 }
-// TODO: bit ops for keys
```

Comment:
I understood this to mean "checking for more complex combinations of keys", but happy to leave this in if that wasn't the intent.